### ISSUE TITLE
Any plan to drop jQuery for just pure Vue JS ?

### ISSUE THREAD

**Issue by darkylmnx** [johnleider/vue-materials#55]
Hi, is there any plan to drop jQuery or do a Vue JS only version ? Because for simple projects it seems a bit overkill to have jQuery imported i think.

**Comment by darkylmnx** [comment]
@johnleider the problem of vuetify or other libs i saw out there is that it doesn't use the materialize css stylesheet and by that mean it's not possible to use the same styles for a page using vue and a static page in the same project. With vue-materials, you can beacause you just need to import the materialize stylesheet via sass so you can keep the same custom styles for pages using view and pages that aren't. The only problem here is jQuery

**Comment by darkylmnx** [comment]
@johnleider and vuetify doesn't use sass wich is disturbing for customisation

**Comment by johnleider** [comment]
Well think about it. If this was to not use jQuery, then that means I would have to write custom javascript for all of that logic. Then I would have to maintain it against their future updates. As far as not using sass, fair point. I'd argue doing simple style customization's with stylus is not too far off from sass.

**Comment by darkylmnx** [comment]
@johnleider fair point for the jQuery question (and for stylus, though i'm more a sass guy), so my next question is, how would i do using vutify to add cutome colors, i mead like really adding my custom color over the current color like i would do on bulma or bootstrap ? Then, how would i do to use the buttons styles on a static page where : i DO NOT use vue but that is part of the same app ?

**Comment by darkylmnx** [comment]
@johnleider just like on the components/buttons page of http://johnleider.com/vue-materials-docs/#! i can simply add the color class to the button for it to please what i want this works on component like on normal html

**Comment by johnleider** [comment]
Excellent question. There are a few ways that you can apply a custom color. For starters, you have access to the entire MD color pallet with custom classes. For changing it at the base, you can utilize one of Vuetify's custom templates and simply updated the theme array. An example can be found [here](https://github.com/vuetifyjs/docs/blob/master/src/stylus/settings/_variables.styl) ,

**Comment by darkylmnx** [comment]
okay thanks, i'll try this and see where it leads me, though the stylus will really be a pain in the ass haha. Thanks.
